packages/relgen-core/src/services/context/remote/types.ts

```diff
@@ -55,7 +55,9 @@ export type RemoteContextService = {
       repo: string;
       num: number;
       excludedFiles?: Set<string>;
-      excludedContexts?: Set<'ticket' | 'file-content'>;
+      excludedContexts?: {
```

Comment:
The export of top-level functions that might have dependencies on other modules is not allowed. Please consider using a Dependency Injection (DI) pattern instead.